crypto/x509/x509_req.c

```diff
@@ -56,6 +56,7 @@ X509_REQ *X509_to_X509_REQ(X509 *x, EVP_PKEY *pkey, const EVP_MD *md)
     }
     return ret;
  err:
+    OPENSSL_free(ri->version->data);
     X509_REQ_free(ret);
```

Comment:
I do not think this is correct. The X509_REQ_free() should free the embedded X509_REQ_INFO and the version, and the data.

Reply:
At least the manual says: `B<I<TYPE>_free>() releases the object and all pointers and sub-objects within it.`